inst/stan/power_multi_ind.stan

```diff
@@ -0,0 +1,149 @@
+// Power law model with multiple individuals
+
+functions{
+  //DE function for use with Runge-Kutta method
+  //pars = [ind_coeff, ind_power]
+  real DE(real y, real y_bar, vector pars){
+    return pars[1] * pow((y/y_bar), -pars[2]);
+  }
+
+  real rk4_step(real y, real y_bar, vector pars, real interval){
+    real k1;
+    real k2;
+    real k3;
+    real k4;
+    real y_hat;
+
+    k1 = DE(y, y_bar, pars);
+    k2 = DE(y+interval*k1/2.0, y_bar, pars);
+    k3 = DE(y+interval*k2/2.0, y_bar, pars);
+    k4 = DE(y+interval*k3, y_bar, pars);
+
+    y_hat = y + (1.0/6.0) * (k1 + 2.0*k2 + 2.0*k3 + k4) * interval;
+
+    return y_hat;
+  }
+
+  real rk4(real y, real y_bar, vector pars, real interval, real step_size){
+    int steps;
+    real duration;
+    real y_hat;
+    real step_size_temp;
+
+    duration = 0;
+    y_hat = y;
+
+    while(duration < interval){
+      //Determine the relevant step size
+      step_size_temp = min([step_size, interval-duration]);
+
+      //Get next size estimate
+      y_hat = rk4_step(y_hat, y_bar, pars, step_size_temp);
+
+      //Increment observed duration
+      duration = duration + step_size_temp;
+    }
+
+    return y_hat;
+  }
+}
+
+// Data structure
+data {
+  real step_size;
+  int n_obs;
+  int n_ind;
+  real y_obs[n_obs];
+  int obs_index[n_obs];
+  real time[n_obs];
+  int ind_id[n_obs];
+  real y_bar[n_ind];
```

Comment:
Sorry if this wasn't clear. This should no vary by individuals, it's a single value applying to the whole dataset